src/layouts/navbar/NavbarItems.jsx

```diff
@@ -0,0 +1,8 @@
+export const NavbarItems = [
```

Comment:
create this types of dummy array in data folder, create data folder in root and inside it create index.js